dependencies.props

```diff
@@ -1,13 +1,13 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
   <PropertyGroup>
-    <CoreFxCurrentRef>61076147ca863b39fc5d32253f31d889ccfd1d12</CoreFxCurrentRef>
+    <CoreFxCurrentRef>3974bbe4abba2327b3b4f063331bc1b760870211</CoreFxCurrentRef>
     <CoreClrCurrentRef>fcd99ed86a703a16698fcd0027707739c8ef3501</CoreClrCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25319-01</CoreFxExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview2-25319-04</CoreFxExpectedPrerelease>
```

Comment:
We missed changing this back to preview1 https://github.com/dotnet/corefx/blob/master/Packaging.props#L3. I'm going to fix that now.